packages/react-card/Spec.md

```diff
@@ -0,0 +1,403 @@
+| Link element                                                                                                               | DocumentCardLocation |                   | ❌        |
+| Extra text component with a grayed out background                                                                          | DocumentCardStatus   |                   | ❌        |
+| Header of the card                                                                                                         |                      | CardHeader        | ❌        |
+| Body of the card                                                                                                           |                      | CardBody          | ❌        |
+| Footer of the card                                                                                                         |                      | CardFooter        | ❌        |
+| Wrapper to render in column format                                                                                         |                      | CardColumn        | ❌        |
+| Allows for content to expand on hover                                                                                      |                      | CardExpandableBox | ❌        |
+
+...
+
+## API Proposal
+
+### Components
+
+| Component   | Purpose                                                                  |
+| ----------- | ------------------------------------------------------------------------ |
+| Card        | Wrapper for the main content of a card.                                  |
+| CardHeader  | Optional header slot for the card.                                       |
+| CardBody    | Main slot of the Card                                                    |
+| CardPreview | Previewing the document/article presented in the card through an image   |
+| CardFooter  | Optional footer slot for the card. (non-applicable for horizontal cards) |
+
+### Card
+
+#### Anatomy
+
+![visual anatomy of the Card component](./assets/Card.png)
+
+#### API
+
+| Property    | Values                                                                               | Default    | Purpose                                                                                   |
+| ----------- | ------------------------------------------------------------------------------------ | ---------- | ----------------------------------------------------------------------------------------- |
+| orientation | `vertical`, `horizontal`                                                             | `vertical` | Orientation of the card                                                                   |
+| size        | `smallest`, `smaller`, `small`, `medium`, `large`                                    | `medium`   | Define the minimum size of the card. Smaller sizes only apply to horizontal card          |
```

Comment:
Shouldn't we also have `larger` and `largest` if we have `smaller` and `smallest` as possible sizes?

Reply:
Design does not differentiate between `large`, `larger` and `largest`. Having them would have no implementation difference.